backend/.env.example

```diff
@@ -1,8 +0,0 @@
-# Example environment variables for backend. Any "<example>"s should be
-# replaced with the real values.
-PORT=3001
-MONGODB_URI="mongodb://127.0.0.1:27017/pap"
-FRONTEND_ORIGIN="http://localhost:3000"
-EMAIL_USER="<example>@gmail.com"
-EMAIL_APP_PASSWORD="<example>"
-BACKEND_FIREBASE_SETTINGS='<example>'

```

Comment:
Is there a reason you guys deleted the `backend/.env.example` file? If not, can we restore this file?